src/models/disassemblyoutput.cpp

```diff
@@ -43,15 +43,24 @@ static QVector<DisassemblyOutput::DisassemblyLine> objdumpParse(const QByteArray
         if (asmLine.isEmpty() || asmLine.startsWith(QLatin1String("Disassembly")))
             continue;
 
+        const int colonIndex = asmLine.indexOf(QLatin1Char(':'));
+        const int angleBracketIndex = asmLine.indexOf(QLatin1Char('<'));
+
+        // skip function header line as it is not interesting for us
+        if (angleBracketIndex > 0 && colonIndex > angleBracketIndex) {
+            continue;
+        }
+
+        // we don't care about the file name
+        if (asmLine.startsWith(QLatin1Char('/')) && asmLine.indexOf(QStringLiteral("file format")) > 0) {
+            continue;
+        }
+
         const auto asmTokens = asmLine.splitRef(QLatin1Char(':'));
         const auto addrLine = asmTokens.value(0).trimmed();
 
         bool ok = false;
         const auto addr = addrLine.toULongLong(&ok, 16);
-        if (!ok) {
```

Comment:
this should still be included, there may be other unexpected lines in the future after all

Reply:
in that case I would suppose to add a regex like this `^   [0-9a-f]{4,}:       [0-9a-f ]{2,}` to verify that we indeed have a disassembled line here to before trying this since source code lines would break this

Reply:
ideally we would extract the source code lines and show them too